clearwater-etcd/usr/share/clearwater/conf/clearwater-etcd.monit

```diff
@@ -46,10 +46,12 @@ check process etcd_process with pidfile /var/run/clearwater-etcd.pid
   # Monit will raise an alarm when it restarts the process
   if memory > 80% for 6 cycles then exec "/bin/bash -c '/usr/share/clearwater/bin/issue_alarm.py monit 6500.3; /etc/init.d/clearwater-etcd stop'"
 
-  # Clear any alarms if the process has been running for 30 seconds
-  if uptime < 30 seconds
-     then exec "/bin/true"
-     else if succeeded then exec "/usr/share/clearwater/bin/issue_alarm.py monit 6500.1"
+# Clear any alarms if the process has been running long enough.
+check program etcd_uptime with path "/usr/share/clearwater/bin/check-uptime /var/run/clearwater-etcd.pid"
```

Comment:
The pidfile path has changed since this PR was made - can you fix this up to /var/run/clearwater-etcd/clearwater-etcd.pid